frontend/packages/dev-console/src/components/import/route/CreateServerlessRoute.tsx

```diff
@@ -0,0 +1,19 @@
+import * as React from 'react';
+import { FormGroup } from 'patternfly-react';
+import { InputField } from '../../formik-fields';
+
+const CreateServerlessRoute: React.FC = () => {
+  return (
+    <FormGroup>
+      <InputField
+        type="text"
+        name="route.targetPort"
```

Comment:
This is still the same form value. `route.targetPort` is shared across both flows and the value is being lost when toggled and the value is changed in the other flow.

Reply:
I see your point now. I have fixed this.